app/utils/discovery_utils.rb

```diff
@@ -0,0 +1,43 @@
+module DiscoveryUtils
+
+  module_function
+
+  def format_params(original)
```

Comment:
I'm thinking about the naming here. To me it's not obvious what DiscoveryUtils contains and less so what DiscoveryUtils::format_params does. Format what params? How? Is this generic or something very specific to this one search query? Do we need this functionality across controllers?